chain/client/src/client.rs

```diff
@@ -581,6 +582,8 @@ impl Client {
         next_height: BlockHeight,
         shard_id: ShardId,
     ) -> Result<Option<(EncodedShardChunk, Vec<MerklePath>, Vec<Receipt>)>, Error> {
+        let _timer = metrics::PRODUCE_CHUNK_TIME.start_timer();
```

Comment:
can we pass a shard_id to this metric? (as chunk production times might get quite different for different shards)